backend/requirements/base.in

```diff
@@ -18,6 +18,7 @@ mozilla-django-oidc-db[setup-configuration]
 django-privates
 django-setup-configuration
 django-health-checks @ git+https://github.com/maykinmedia/django-health-checks.git@99b5ce57917abc40e16543b7fb251ba710524b31
+django-csp
```

Comment:
I guess upgrading to Django 6 (offers built-in CSP settings) is too enthusiastic

Reply:
I thought about it too, but I think that would be a different issue.